apps/payments/next/app/[locale]/[offeringId]/[interval]/new/route.ts

```diff
@@ -49,10 +68,10 @@ export default async function CheckoutNew({
     }
   }
 
-  const searchParamsString = new URLSearchParams(searchParams).toString();
-  if (searchParamsString) {
-    redirect(`${cartId}/start?${searchParamsString}`);
-  } else {
-    redirect(`${cartId}/start`);
-  }
+  redirectToUrl.pathname = redirectToUrl.pathname.replace(
+    '/new',
+    `/${cartStatus}/${cartId}/start`
+  );
+
+  redirect(redirectToUrl.href);
```

Comment:
Example of `redirectToUrl.href`
`http://localhost:3035/en/123donepro/monthly/checkout/6cb5cec5fd3941c38ed3858c3b869d95/start?_rsc=1wtp7`

Reply:
Initial `redirectToUrl`
```
9|payments-next  | NextURL {
9|payments-next  |   [Symbol(NextURLInternal)]: {
9|payments-next  |     url: URL {
9|payments-next  |       href: 'http://localhost:3035/en/123donepro/monthly/new?_rsc=1wtp7',
9|payments-next  |       origin: 'http://localhost:3035',
9|payments-next  |       protocol: 'http:',
9|payments-next  |       username: '',
9|payments-next  |       password: '',
9|payments-next  |       host: 'localhost:3035',
9|payments-next  |       hostname: 'localhost',
9|payments-next  |       port: '3035',
9|payments-next  |       pathname: '/en/123donepro/monthly/new',
9|payments-next  |       search: '?_rsc=1wtp7',
9|payments-next  |       searchParams: URLSearchParams { '_rsc' => '1wtp7' },
9|payments-next  |       hash: ''
9|payments-next  |     },
9|payments-next  |     options: { headers: [Object], nextConfig: undefined },
9|payments-next  |     basePath: '',
9|payments-next  |     domainLocale: undefined,
9|payments-next  |     defaultLocale: undefined,
9|payments-next  |     buildId: undefined,
9|payments-next  |     locale: undefined,
9|payments-next  |     trailingSlash: false
9|payments-next  |   }
9|payments-next  | }
```

After replace
```
9|payments-next  | NextURL {
9|payments-next  |   [Symbol(NextURLInternal)]: {
9|payments-next  |     url: URL {
9|payments-next  |       href: 'http://localhost:3035/en/123donepro/monthly/checkout/0fc64428d449406799c3817d7b23554e/start?_rsc=1wtp7',
9|payments-next  |       origin: 'http://localhost:3035',
9|payments-next  |       protocol: 'http:',
9|payments-next  |       username: '',
9|payments-next  |       password: '',
9|payments-next  |       host: 'localhost:3035',
9|payments-next  |       hostname: 'localhost',
9|payments-next  |       port: '3035',
9|payments-next  |       pathname: '/en/123donepro/monthly/checkout/0fc64428d449406799c3817d7b23554e/start',
9|payments-next  |       search: '?_rsc=1wtp7',
9|payments-next  |       searchParams: URLSearchParams { '_rsc' => '1wtp7' },
9|payments-next  |       hash: ''
9|payments-next  |     },
9|payments-next  |     options: { headers: [Object], nextConfig: undefined },
9|payments-next  |     basePath: '',
9|payments-next  |     domainLocale: undefined,
9|payments-next  |     defaultLocale: undefined,
9|payments-next  |     buildId: undefined,
9|payments-next  |     locale: undefined,
9|payments-next  |     trailingSlash: false
9|payments-next  |   }
9|payments-next  | }
```